docs/src/main/paradox/bootstrap/kubernetes.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes
+
+As of Akka 2.5.15 and Akka-Management 0.18 the recommended way to run Akka Cluster in Kubernetes is to:
+
+* Use Akka Bootstrap with `akka-dns` with cluster formation via DNS SRV records
+* Use a headless service for internal for Akka management/bootstrap so that readiness probes for prod traffic don't interfere with bootstrap 
+* If required use a separate service and/or ingress for user facing endpoints e.g. HTTP gRPC
```

Comment:
Maybe add some more punctiation, like "ingress for user-facing endpoints, for example HTTP or gRPC"? Maybe add links to akka-http and akka-grpc? Do we know if ingress works for gRPC?

Reply:
I haven't tried ingress for gRPC only internal comms. It'll depend on what is your ingress controller, most of them are nginx